jujugui/static/gui/src/app/app.js

```diff
@@ -1342,13 +1344,20 @@ YUI.add('juju-gui', function(Y) {
       @method _renderProviderLogo
     */
     _renderProviderLogo: function() {
-      const cloudProvider = this.env.get('providerType');
+      const containerId = 'provider-logo-container';
+      const clearLogo = () => {
+        document.getElementById(containerId).innerHTML = '';
```

Comment:
Because it's a react component it should probably be removed with https://facebook.github.io/react/docs/react-dom.html#unmountcomponentatnode